addons/PseudoCode_Console/src/presenter.js

```diff
@@ -60,7 +60,7 @@
 /******/ 	__webpack_require__.p = "";
```

Comment:
wywal zmiany w tym pliku. Generalnie pilnuj, żeby commitować tylko to, co rzeczywiście chcesz zacommitować. W szczególności unikaj komend typu git add * , chyba, że wcześniej potwierdziłeś, że wszystko co wejdzie do commit'a powinno do niego wejść.